client/src/components/EventView/Header/index.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FiPrinter, FiDownload } from "react-icons/fi";
+import styled from "styled-components";
+
+import IconButton from "../../Common/IconButton";
+import InputEditable from "../../Common/InputEditable";
+import DateEditable from "../../Common/DateEditable";
+import Stack from "../../Common/Stack";
+import {
+  theme1,
+  fontSize64,
+  fontSize24,
+  spacing8,
+  spacing16,
+  sideMargins
+} from "../../../resources/style-constants";
+
+const Container = styled.div`
+  background-color: ${(props) => props.bgcolor || theme1};
+  padding: 0 ${sideMargins};
+`;
+
+const TitleContainer = styled.div`
+  width: 60%;
```

Comment:
For this component, we wouldn't want to set a width. If the event has a super long title, we would want it to extend to the other margin, and then wrap if necessary.